doc/dev/radosgw/s3_compliance.rst

```diff
@@ -290,7 +290,7 @@ S3 Documentation reference : http://docs.aws.amazon.com/AmazonS3/latest/API/REST
 +---------+---------------------------+------------+---------------------------------------------------------------------------------------------------------+-------------+
 | PUT     | Object copy               | Yes        |                                                                                                         |             |
 +---------+---------------------------+------------+---------------------------------------------------------------------------------------------------------+-------------+
-| PUT     | Initate multipart upload  | Yes        |                                                                                                         |             |
+| PUT     | Initiate multipart upload | Yes        |                                                                                                         |             |
```

Comment:
can you verify the table still renders properly?

Reply:
I was expecting to find the documentation as a CI job artefact, but I cannot find anything in the [Docs: build check](https://jenkins.ceph.com/job/ceph-pr-docs/57704/) CI job.

Reply:
Got it it's the [docs/readthedocs.org:ceph](https://ceph--44249.org.readthedocs.build/en/44249/) job instead.

Points to the documentation here:
https://ceph--44249.org.readthedocs.build/en/44249/

Reply:
The table renders as expected, as far as I can see:
https://ceph--44249.org.readthedocs.build/en/44249/dev/radosgw/s3_compliance/#operations-on-objects